#!/usr/bin/env python
PACKAGE = "custom_social_layer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("stretch_factor", int_t, 0, "Factor by which velocity stretches proxemics (meters/meters/second", 1, 0, 100)
gen.add("enabled", bool_t, 0, "Whether to apply this plugin or not", True)
gen.add("ellipse_width",  int_t, 0, "Side to side proxemics (meters)", 1)
gen.add("ellipse_height", int_t, 0, "Front-back proxemics (meters)",   2)

exit(gen.generate(PACKAGE, "custom_social_parameters", "PeopleParameters"))
